### PR TITLE
feat(website/homepage): new homepage integration

### PR DESCRIPTION
**Pull request by tiloyi**
## I have read [the contributing guidelines](https://mozaic.adeo.cloud/Contributing/)

- [x] Yes
- [ ] No

## Does this PR introduce a [breaking change](https://mozaic.adeo.cloud/Contributing/Developers/GitConventions/#breaking-changes-)?

- [ ] Yes
- [x] No

## Describe the changes

Homepage redesign integration

GitHub issue number or Jira issue URL: [Card-9006](https://adeo.kanbanize.com/ctrl_board/10/cards/9006/details/)

### CODE REVIEW

File: src/components/Header/Header.js [hunk 1]
```diff
@@ -0,0 +1,206 @@
...
+        </HeaderDesc>
+      </HeaderMain>
+      <HeaderAside className="ml-flexy__col ml-flexy__col--full ml-flexy__col--1of3@from-l">
+        <HeaderSubtitle>Play with Mozaic</HeaderSubtitle>
+        <div className="header__playground">
+          <StyledMcButton label="Button label" />
+          <StyledMcTextInput />
+          <StyledMcCheckbox label="Checkbox Label" />
+          <HeaderWrapper>
+            <McToggle size="s" label="Small" />
```
Comment (nherwegh) [inline src/components/Header/Header.js:191]: can we have the toggles without label?

File: src/components/Header/Header.js [hunk 2]
```diff
@@ -0,0 +1,206 @@
...
+      </a>
+    </HeaderTop>
+    <div className="ml-flexy ml-flexy--gutter">
+      <HeaderMain className="ml-flexy__col ml-flexy__col--push-1of12@from-l">
+        <HeaderTitle>Welcome to Mozaic Design System</HeaderTitle>
+        <HeaderDesc>
+          A design system to reinforce team’s delivery time, improve quality of
+          deliverables and thus help set up a coherent customer experience.
+        </HeaderDesc>
+      </HeaderMain>
```
Comment (nherwegh) [inline src/components/Header/Header.js:183]: can we have more blank space here? We have one empty columns between the text and the pattern previews on the design

File: src/components/AreaValues/AreaValues.js [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+      index={`#${index + 1}`}
+      title={block.title}
+      text={block.text}
+    />
+  ))
+
+  return (
+    <AreaContainer>
+      <AreaTitle>Our values</AreaTitle>
+      <div className="ml-flexy ml-flexy--gutter">{blocks}</div>
```
Comment (nherwegh) [inline src/components/AreaValues/AreaValues.js:82]: the font-weight for the numbers is `regular` here

File: src/components/Header/Header.js (hunk 1 above)
Comment (tiloyi) [reply src/components/Header/Header.js:191]: Unfortunately, currently in the documentation it is not indicated that this component can be used without its label. As a result, the code is not designed to handle this case.

Would you like us to make an evolution on this subject ?

File: src/components/Header/Header.js (hunk 2 above)
Comment (tiloyi) [reply src/components/Header/Header.js:183]: fixed

File: src/components/AreaValues/AreaValues.js (hunk 3 above)
Comment (tiloyi) [reply src/components/AreaValues/AreaValues.js:82]: fixed

File: src/components/Header/Header.js (hunk 1 above)
Comment (nherwegh) [reply src/components/Header/Header.js:191]: @caqueste what do you think for that?
maybe we can keep the toggles like that.

**Review by mohamedMok — APPROVED** [review]

**Review by nherwegh — APPROVED** [review]

**Review by caqueste — APPROVED** [review]

**Pull request merged**